lib/index.js

```diff
@@ -179,6 +200,39 @@ FSTree.defaultIsEqual = function defaultIsEqual(entryA, entryB) {
   return equal;
 };
 
+FSTree.applyPatch = function(input, output, patches, _delegate) {
```

Comment:
Left it as `applyPatch` to match `calculatePatch`. I named the argument `patches` though so that is is apparent it is an array. Probably just need to disambiguate the term "patch".

Perhaps it could be `patch` represent the array and each item is an `operation`?